src/wp-admin/about.php

```diff
@@ -26,11 +26,11 @@
 		<p class="about-text">
 			<?php printf(
 				/* translators: link to "business-focused CMS" article */
-				__( 'Thank you for using ClassicPress, the <a href="%s">business-focused CMS</a>.' ),
+				__( 'Thank you for using ClassicPress, the <a href="%s">CMS for Creators</a>.' ),
 				'https://www.classicpress.net/blog/2018/10/29/classicpress-for-business-professional-organization-websites/'
```

Comment:
This link needs an update. I think we can set it to https://link.classicpress.net/the-cms-for-creators and have that redirect to https://www.classicpress.net for now.

Reply:
Recommendation noted.